templates/.circleci/config.yml

```diff
@@ -231,11 +231,23 @@ jobs:
       - run:
           name: Deploy
           command: |
-            kubectl create namespace << parameters.namespace >> || echo "Namespace already exists"
+            DEPLOYMENT=<< parameters.repo >>
```

Comment:
I didn't declare another variable, since we use <% .Name %> for both the deployment name and ecr-repo, should i declare it just for extra clarity?

Reply:
It's maybe a little less reusable, but it should be fine.